.github/CONTRIBUTING.md

```diff
@@ -171,7 +171,7 @@ Run these commands
 git submodule update --init --recursive
 pip install -r requirements/docs.txt
 cd docs
-make html
+make html  # You could disable copying notebooks by running `SHOULD_COPY_NOTEBOOKS=0 make html`
```

Comment:
still not working for me.. have tried `PL_SHOULD_COPY_NOTEBOOKS=0 make html` too.

Reply:
Same here. It exits with the error:
```console
$ PL_SHOULD_COPY_NOTEBOOKS=0 make html
...
Extension error (pt_lightning_sphinx_theme.extensions.lightning_tutorials):
Handler <function process_card_nodes at 0x7f4c51ca3790> for event 'doctree-resolved' threw an exception (exception: 'BuildEnvironment' object has no attribute 'all_cardnodes')
make: *** [Makefile:19: html] Error 2
```

Reply:
@rohitgr7 Pushed a commit 537114c to fix this error. Can you try again if you get a chance? Although it raises some warnings, the docs build was successful in my local env.

Reply:
still not working. I'll ping you on slack